.github/workflows/thoth.yml

```diff
@@ -0,0 +1,69 @@
+name: THOTH
+
+on: [push, pull_request]
+
+jobs:
+  thoth:
+    runs-on: ubuntu-latest
+    env:
+      CI_COMMIT_MESSAGE: CI Generating contract's call graphs
+      CI_COMMIT_AUTHOR: ${{ github.event.repository.name }} CI
+    steps:
+      # Python setup
+      - name: Set up Python 3.10
```

Comment:
we use python 3.9 in the project, is this on purpose?

Reply:
while trying to create the action it failed with 3.9.
As I faced many issue to do this action trying python 3.10 was something I tried, and I forgot to revert it, will change it back.